components/brave_ads/core/internal/resources/behavioral/conversions/conversions_info.h

```diff
@@ -6,31 +6,22 @@
 #ifndef BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_RESOURCES_BEHAVIORAL_CONVERSIONS_CONVERSIONS_INFO_H_
 #define BRAVE_COMPONENTS_BRAVE_ADS_CORE_INTERNAL_RESOURCES_BEHAVIORAL_CONVERSIONS_CONVERSIONS_INFO_H_
 
-#include <memory>
 #include <string>
 
+#include "base/types/expected.h"
+#include "base/values.h"
 #include "brave/components/brave_ads/core/internal/resources/behavioral/conversions/conversion_id_pattern_info.h"
 
-namespace base {
-class Value;
-}  // namespace base
-
 namespace brave_ads::resource {
 
 struct ConversionsInfo final {
   ConversionsInfo();
-
-  ConversionsInfo(const ConversionsInfo& other);
```

Comment:
Can we please `= delete` rather than removing

Reply:
We can, but again, rule of five is not part of the style guide:

https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types

I should remind you again that we shouldn't have silos in the codebase that have their own little styles, and things of this type should be to the purview of the engineer.

Reply:
Done. Followed up on slack for more details.

Reply:
I don't have a strong opinion regarding this, but Google code style guide doesn't deny the rule of five.
Another reason for following the rule of five here is that it is consistent with the existing codebase in Brave Ads. 
See `Be consistent with existing code` paragraph: https://google.github.io/styleguide/cppguide.html#Goals